test/factories/factories.rb

```diff
@@ -13,16 +13,19 @@
     title { FactoryGirl.generate(:project_title) }
     description " "
     total 40
+    status 0
+    created_at Time.now
```

Comment:
Do not use `Time.now` without zone. Use one of `Time.zone.now`, `Time.now.current`, `Time.now.in_time_zone`, `Time.now.utc`, `Time.now.getlocal`, `Time.now.iso8601`, `Time.now.jisx0301`, `Time.now.rfc3339`, `Time.now.to_i`, `Time.now.to_f` instead.